src/main/python/snakepit/main.py

```diff
@@ -56,6 +57,13 @@ class TemplateNoteFoundException(Exception):
     pass
 
 
+class OutputDirectoryNotWritable(Exception):
```

Comment:
Also, if you write code to check this, you should probably test it as part of the cram tests.